src/occa/internal/lang/expr/dpcppAccessorNode.cpp

```diff
@@ -0,0 +1,69 @@
+#include <occa/internal/lang/expr/dpcppAccessorNode.hpp>
+#include <occa/internal/utils/string.hpp>
+
+namespace occa
+{
+  namespace lang
+  {
+
+    dpcppAccessorNode::dpcppAccessorNode(token_t *token_,
+                                         const vartype_t &shared_type_,
+                                         const std::string &handler_name_)
+        : exprNode{token_}, shared_type{shared_type_}, handler_name{handler_name_}
```

Comment:
Nit: We aren't using the `{}` intializers, but it doesn't seem to impact anything either so might be ok to keep
```suggestion
        : exprNode(token_), shared_type(shared_type_), handler_name(handler_name_)
```